### DISCUSSION TITLE
How do i pull the response from an http request?

### DISCUSSION THREAD

**Discussion by Ferulaindig** [DeabLabs/cannoli#15] [Q&A] [answered]
I'm not very knowledgeable about making rest calls so bear with me. I'm attempting to send a webhook message to a discord channel. Am getting a 400 response. I was hoping the response body might contain information about what is responsible for the error

**Comment by blindmansion** [comment]
Yea that would make a lot more sense. Currently when you get a 400 error cannoli creates a red error node correct?

I think better behavior would be to send the error forward, like you said, and just give a warning.

**Reply by Ferulaindig** [reply]
Yeah i believe it's just a red card that popups. I do like the red popup card. Maybe a new type of popup, for when it's possible to continue execution after error. I'm not sure if error continues after execution with the current version. Don't know enough about the plugin currently to know if this is already a thing. Feel free to let me know otherwise. Then like you had mentioned pass the error on.

**Reply by blindmansion** [reply, marked as answer]
Yes there are "warning" nodes, which are also red, but they don't halt the whole cannoli. I'll take a look at the current error handling for action nodes and move towards that behavior. Thanks for the suggestion!
